### PR TITLE
Exapand rationale against AES-GCM-SIV

### PR DESCRIPTION
**Pull request by expede**
[Preview 👀](https://github.com/wnfs-wg/spec/blob/expand-no-siv/rationale/encryption.md)

### CODE REVIEW

File: .github/workflows/words-to-ignore.txt [hunk 1]
```diff
@@ -1,106 +1,108 @@
 # Names
```
Comment (expede) [inline .github/workflows/words-to-ignore.txt:1]: I had to add deduplication to this list (weird!) I also sorted it while I was in here to make stuff easier to find.

Comment (matheus23) [reply .github/workflows/words-to-ignore.txt:1]: Thank you! :)

**Review by matheus23 — APPROVED** [review]
Great summary :) I'm glad that we have the rationale files :grin:

File: .github/workflows/words-to-ignore.txt (hunk 1 above)
Comment (matheus23) [reply .github/workflows/words-to-ignore.txt:1]: Yeah. If I write "deduplication" in this box, my browser also complains.
The spellchecker is using some standard american-english dictionary for hunspell. I wish there was a "programmer jargon"-specific dictionary though.

Comment (expede) [reply .github/workflows/words-to-ignore.txt:1]: It's fine. This will all get replaced by GPT eventually 😅

Comment (matheus23) [reply .github/workflows/words-to-ignore.txt:1]: *squinting*

That's actually not a bad idea. Makes me want to pull out some open-source LLM and attach it to my spellchecker. That's a self-nerd-snipe for another day though :grin:

**Pull request merged**